packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -496,6 +508,7 @@ export class SmoothControls extends EventDispatcher {
     if (TOUCH_EVENT_RE.test(event.type)) {
       const {touches} = event as TouchEvent;
 
+      // tslint:disable-next-line:switch-default
```

Comment:
I try to avoid these where possible; what is this about?

Reply:
tslint is complaining that there is no 'default:' case here. It looks like if no case label is matched the behavior is to fall out of the switch statement and execute the line 'this.lastTouches = touches;'

I will just add a default statement that breaks out of the switch, a bit of future proofing.